react/MuiCozyTheme/Divider/Readme.md

```diff
@@ -27,3 +27,19 @@ import Grid from 'cozy-ui/transpiled/react/MuiCozyTheme/Grid';
   <Typography variant="body1" className="u-ml-1">Right block</Typography>
 </Grid>
 ```
+
+Divider with text (left)
+
+```jsx
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider';
+
+<Divider>Text Left</Divider>
+```
+
+Divider with text (center)
+
+```jsx
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider';
+
+<Divider text="center">Text Center</Divider>
```

Comment:
The API is not very clear. If I quickly read it, I expect that "center" will be displayed since this is the text of the Divider. And having the "Text Center" displayed somewhere. 

What do you think about textAlign="center"?

Reply:
Also, If I understand correctly `textAlign` is only valid for TextDivider. We should add the proptypes to describe this behavior

Reply:
Yea I agree "text" can be confused as an "input" prop. I was trying to follow Figma naming convention though. Should I add a css class for textAlign="left" even if it's the default value? (right align is not in Figma)

Reply:
@Crash-- I'm not sure where to put that proptype, in TextDivider or in the Divider dispatcher? (so I will probably add a new proptype to the mui proptype object)

Reply:
update: mui doesn't use proptypes but typescript

Reply:
This is not for MUI, this is for our styleguidist! See https://docs.cozy.io/cozy-ui/react/#!/Badge for exemple.